frontend/src/components/common/AccessDropdown/index.tsx

```diff
@@ -0,0 +1,174 @@
+// Libraries
+import React, { useEffect, useMemo, useState } from 'react';
+
+// Components
+import { Modal, Input, List, Select, message, Tag, AutoComplete, Typography } from '../../ui';
+import { AddIcon, CloseIcon, DeleteIcon, SwitchUserIcon } from '../../icons';
+
+// Constants
+import { PERMISSION, ROLE_KEY, ROLE_OPTIONS } from 'constants/role';
+import {
```

Comment:
@Btrnn Wrong position, should put in // Services group

Reply:
@irisnikita i fixed, check again.

Reply:
@Btrnn Oke